src/Models/EvidenceItem.php

```
@@ -10,30 +10,41 @@ final class EvidenceItem
+                    ':title' => (string) $data['title'],
+                    ':evidence_snippet' => (string) ($data['evidence_snippet'] ?? ''),
+                    ':location_hint' => $data['location_hint'] ?? null,
+                    ':actor_label' => $data['actor_label'] ?? null,
+                    ':confidence' => $data['confidence'] ?? null,
+                    ':review_status' => $data['review_status'] ?? 'proposed',
+                    ':metadata_json' => self::jsonOrNull($data['metadata_json'] ?? null),
+                ]
+            );
+
+            if ($stmt->rowCount() === 0) {
+                throw new \RuntimeException('Project does not belong to organisation');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '43', column '27'). (undefined)

(MissingImport)